powersimdata/input/check.py

```diff
@@ -210,3 +222,362 @@ def _check_connected_components(grid, error_messages):
             f"but is specified as having {num_interconnects} interconnects: "
             f"{grid.interconnect}."
         )
+
+
+def _check_data_frame(df, label):
+    """Ensure that input is a pandas data frame.
```

Comment:
Inspired by @rouille 's comments below, do we want an empty line here after the description as well? Same comments for the rest functions in this module.

Reply:
Done.